.github/workflows/continuous_integration.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         python -m build
     - name: Test with tox
       run: |
-        python -m pip install tox tox-gh-actions
+        python -m pip install tox tox-gh-actions setuptools
```

Comment:
setuptools installation is no longer needed after #1183 , you can revert this line.

Reply:
```suggestion
        python -m pip install tox tox-gh-actions
```